.github/workflows/cypress.yml

```diff
@@ -0,0 +1,162 @@
+name: Cypress
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'appinfo/**'
+      - 'cypress/**'
+      - 'l10n/**'
+      - 'lib/**'
+      - 'src/**'
+      - 'templates/**'
+      - '.php-cs-fixer.dist.php'
+      - 'babel.config.js'
+      - 'composer.json'
+      - 'composer.lock'
+      - 'cypress.json'
+      - 'package.json'
+      - 'package-lock.json'
+
+  push:
+    branches:
+      - main
+      - master
+      - stable*
+
+permissions:
+  contents: read
+
+concurrency:
+  group: cypress-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  APP_NAME: monthly_status_email
+  CYPRESS_baseUrl: http://localhost:8081/index.php
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16]
+        php-versions: ['8.0']
+        databases: ['sqlite']
+        server-versions: ['stable25', 'master']
```

Comment:
Newest and latest only? Should be fine from my pov

Reply:
Well, testing against stable25 makes sense because it's supported and testing against master is important to catch breakages :wink: